python/lsst/pipe/supertask/activator.py

```diff
@@ -350,15 +385,16 @@ def get_tasks(modules_only=False):
                     module_classes = [class_key for class_key in classes_map.keys() if
                                       classes_map[class_key].module == module_name]
                     for mod_cls in module_classes:
-                        # Get all task except the base classes
+                        # Get all task except the base classes, they need to be ended with 'Task'
                         if mod_cls[-4:].upper() == 'TASK' and mod_cls not in [
                             'WorkFlowParTask', 'WorkFlowSeqTask', 'Task', 'SuperTask',
                             'WorkFlowTask', 'CmdLineTask']:
```

Comment:
This list of exempted class names should probably be listed in a tuple at the top of the file rather than hiding it in here.